prow/config.yaml

```diff
@@ -322,6 +322,19 @@ branch-protection:
               <<: *release114
             release-1.15:
               <<: *release115
+            experimental-dual-stack:
+              protect: true
+              restrictions:
+                teams:
+                - zhlsunshine
+                - jacob-delgado
+            experimental-ambient:
+              protect: true
+              restrictions:
+                teams:
+                - stevenctl
```

Comment:
@howardjohn need a new team for this?

Reply:
Good question.. I was originally thinking we would just use the standard CODEOWNERS, but maybe we don't want to spam them all.

Although maybe the config.yaml shouldn't have this, and the forks should just modify CODEOWNERS as fit?

Reply:
for ambient branch, maybe the CODEOWNERS know less than people who really involved.

BTW change the CODESOWNERS file in target branch will also bring more stuff when you try merge it into master?